.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,19 @@
+name: Benchmark Pull Requests
```

Comment:
Let's name the workflow simpler, e.g. something like `Run benchmarks`.

Also let's name the file as `benchmark.yml`, because later, the other workflow might be set for PRs, too.